### DISCUSSION TITLE
No highlighting for custom_attributes in title template

### DISCUSSION THREAD

**Discussion by RuslanHryn** [magicbell/community#219] [General]
Put a template to get custom attributes into the title.
For example:

`{{ notification.custom_attributes["some-key"] }}  separator {{ user.custom_attributes["some-key"] }}`

and no values from attributes will be rendered in the title.

No issues if use the same template in the content.

Also, `{{ notification.custom_attributes }}  separator {{ user.custom_attributes }} `will be rendered correctly in the title. The problem is that getting value by keys is not working properly in title only

**Comment by smeijer** [comment]
Hi Ruslan,

The tags do work there; they're just not highlighted. Sorry for the inconvenience. It's been on our radar, and we will fix it rather sooner than later.
